api/RTCRtpSender.json

```
@@ -659,6 +659,351 @@
+                },
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": "12.1"
+                },
+                "safari_ios": "mirror",
+                "samsunginternet_android": "mirror",
+                "webview_android": "mirror"
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "priority": {
+            "__compat": {
+              "description": "<code>parameters.encodings.priority</code> parameter",
+              "spec_url": "https://w3c.github.io/webrtc-priority/#dom-rtcrtpencodingparameters-priority",
+              "support": {
+                "chrome": {
+                  "version_added": "69"
+                },
+                "chrome_android": "mirror",
+                "edge": {
+                  "version_added": "≤79"
+                },
+                "firefox": {
+                  "version_added": "46",
+                  "notes": "In version 110 and later the default priority is <code>low</code>."
+                },
+                "firefox_android": "mirror",
+                "ie": {
+                  "version_added": false
+                },
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": "12.1"
+                },
+                "safari_ios": "mirror",
+                "samsunginternet_android": "mirror",
+                "webview_android": "mirror"
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "rid": {
+            "__compat": {
+              "description": "<code>parameters.encodings.rid</code> parameter",
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "46"
+                },
+                "firefox_android": "mirror",
+                "ie": {
+                  "version_added": false
+                },
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror",
+                "samsunginternet_android": "mirror",
+                "webview_android": "mirror"
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
```

"rid" is not in the spec, so is non-standard
```suggestion
                "standard_track": false,
```

It is only in one browser but for more than two years, so probably "experimental" false is correct.

But it MAY be that it is deprecated - it is in FF but FF is working to match the spec now.

---

I was wrong. It is in the spec, just derived. However it might be experimental given it is only in FF.